README.md

```diff
@@ -1 +1,39 @@
-# Duc_Hieu_Nguyen_JS_PRACTICE
\ No newline at end of file
+## OVERVIEW
+
+- This document provides the list of important technologies that would be used to implement, the objectives to be achieved and the progress of the project.
+
+## TIMELINE:
+
+- 2 week (Apr 20 - May 4)
+
+## TARGET:
+
+- Understand and apply knowledge of HTML/CSS/JavaScript to build website
+- JavaScript ES6 syntax
+- Get familiar with MVC architecture
+
+## TECHNICAL STACKS
+
+- HTML5/CSS3
+- JavaScript
+- Parcel
+- MVC
+
+## WEB DESIGN:
+
+- Later
+
+## SOURCE CODE
+
+- Clone project : git clone git
+
+## HOW TO RUN
+
+- Dillinger requires Node.js v10+ to run.
+- Install the dependencies and devDependencies and start the server.
+
+```
+$npm install --save-dev parcel
+$npm start
+Open on web: http://localhost:1234
```

Comment:
and review this here

Reply:
I think I will update after the project is done

Reply:
I think you should update properly as soon as you initialize Pratice

Reply:
Pls update this file before start coding